### PR TITLE
Feature/df 391 scrolling issue

### PR DESCRIPTION
**Pull request by 99999989**

### CODE REVIEW

File: alv-portal-ui/src/app/shared/layout/three-column-layout/three-column-layout.component.ts [hunk 1]
```diff
@@ -9,12 +9,19 @@ export class ThreeColumnLayoutComponent implements OnInit {
 
   @Input() mobileOrdering: number[];
 
-  constructor() { }
+  @Input() stickyTop = 0;
+
+  private readonly BOOTSTRAP_BREAKPOINT_SM = '(max-width: 767.98px)';
```
Comment (pavax) [inline alv-portal-ui/src/app/shared/layout/three-column-layout/three-column-layout.component.ts]: add comment where this magic 767.98 is copied from

File: alv-portal-ui/src/app/shared/layout/three-column-layout/three-column-layout.component.ts [hunk 2]
```diff
@@ -24,4 +31,25 @@ export class ThreeColumnLayoutComponent implements OnInit {
     return 'order-' + this.mobileOrdering[columnIndex];
   }
 
+  private listenToWindowResize() {
+    window.addEventListener('resize', (event) => {
```
Comment (pavax) [inline alv-portal-ui/src/app/shared/layout/three-column-layout/three-column-layout.component.ts]: remove the eventlistener on destroy

Comment (99999989) [reply alv-portal-ui/src/app/shared/layout/three-column-layout/three-column-layout.component.ts]: done

File: alv-portal-ui/src/app/shared/layout/three-column-layout/three-column-layout.component.ts (hunk 1 above)
Comment (99999989) [reply alv-portal-ui/src/app/shared/layout/three-column-layout/three-column-layout.component.ts]: done

**Pull request merged**